src/views/macros/DefaultMacro.tsx

```diff
@@ -1,9 +1,10 @@
+import React from 'react';
 import type {MacroProps} from '../../types';
 import HTMLReactParser from 'html-react-parser';
 
 
 const DefaultMacro = ({name, config, meta}: MacroProps) => (
-    HTMLReactParser(config.body) as JSX.Element
+    HTMLReactParser(config.body) as React.JSX.Element
```

Comment:
Is there a particular reason you did that ?
Starting from react 17 it should be safe to just use JSX.Element without explicitly importing React at the top.